res/sections/42-ComandiBase.tex

```diff
@@ -0,0 +1,109 @@
+supporti sono \textbf{pdf}, \textbf{png} e \textbf{jpg}. Nelle nuove 
+installazioni di \LaTeX{} è possibile utilizzare però anche il formato \textbf{
+eps}.
+
+Di seguito possiamo vedere un esempio di come inserire un'immagine con 
+didascalia in un documento \LaTeX{}.
+\lstinputlisting{res/examples/esempioimmagine.tex}
+
+
+
+\subsection{Affiancare le immagini}
+(Fate un respiro profondo che sennò qui si perde il proprio posto in paradiso) 
+\\
+Per poter affiancare immagini è possibile utilizzare il \textit{subfig} (Non è 
+l'unica soluzione, ma è una delle più semplici). Questo permette di creare 
+``delle piccole immagini'' all'interno dell'ambiente 
+\verb!\begin{figure}...\end{figure}!. Specificando grandezza delle immagini o 
+utilizzando i comandi per iniziare una nuova riga. È probabilmente più facile 
+con un esempio.
+\lstinputlisting{res/examples/esempio4immagini.tex}
+Nell'esempio viene utilizzato il comando \verb!\subfloat! per posizionare le 
+quattro immagini al quale, tra parentesi quadre, viene specificata la 
+didascalia della singola immagine, mentre tra parentesi graffe viene indicata 
+l'immagine da mostrare. Con il comando \verb!\caption! viene, infine, 
+specificata la didascalia per le immagini raggruppate.
+
+\subsection{Affiancare testo e immagini}
+Sì si può anche affiancare testo e immagini. Serve? Bho vedete voi. In ogni 
+caso è possibile utilizzando il \textit{package} \verb!wrapfig!. Questo 
+funziona in modo analogo al solito modo di inserire le immagini, tranne che 
+l'ambiente sarà \verb!\begin{wrapfig}{pos}{dim}...\end{wrapfig}! dove \texttt{
+pos} può essere \textbf{r} o \textbf{l} e \texttt{dim} serve per specificare 
+le dimensioni di quanto l'immagine deve ``affiancarsi'' al testo. Di seguito un 
+esempio. 
+\lstinputlisting{res/examples/esempiotestoafianco.tex}
+
+
+\section{Note a piè di pagina}
+Spesso capita di dover mettere delle note a piè di pagina, per esempio per mettere un riferimento o spiegare meglio qualcosa\footnote{O solo per distrarre il lettore, come in questo caso}. Per fare ciò è necessario utilizzare il comando \verb!\footnote{...}! specificando tra graffe il testo della nota. \LaTeX{} in automatico si occuperà della numerazione e del posizionamento.
+
+
+\section{Collegamenti ipertestuali}
+
+\subsection{Collegamenti nel testo}
+Quando si inseriscono delle immagini e se ne fa riferimento nel testo oppure quando si scrive una parte e la si riprende in un'altra sezione del documento si sente la necessità di inserire un collegamento tra le due parti. Per fare ciò è necessario marcare con \verb!\label{...}! a cui si vuole fare riferimento. Tra parentesi graffe un identificativo univoco all'interno del documento. Quando ci si riferisce ad esso, si deve utilizzare \verb!\ref{...}!, specificando tra graffe lo stesso identificativo. È possibile anche fare riferimento alla pagina in cui si trova l'oggetto marcato con \verb!label! utilizzando \verb!\pageref{...}! (e lo stesso identificativo).
```

Comment:
Sta parte non è chiara secondo me.

> marcare con \verb!\label{...}! a cui si vuole fare riferimento

volevi dire `marcare con \verb!\label{...}! le parti a cui si vuole fare riferimento`?

> Per fare ciò è necessario marcare con \verb!\label{...}! a cui si vuole fare riferimento. Tra parentesi graffe un identificativo univoco all'interno del documento. Quando ci si riferisce ad esso, si deve utilizzare \verb!\ref{...}!, specificando tra graffe lo stesso identificativo.

C'è la forte possibilità che sia mona io, ma a queste tre frasi lette di seguito non riesco a dargli un senso. Soprattutto alla seconda non riesco a capire proprio cosa vuoi dire, scusami 😅 Boh secondo me è forse meglio riscrivere questi tre punti in maniera diversa 😅 

Anche qui ti sei dimenticato le 80 colonne comunque 😁